source/welcome/whats-new-in-v72.rst

```diff
@@ -100,6 +100,10 @@ Automatic migration
 
 Instead of being tied to a particular channel, each board has been automatically migrated to use its own access control list. The creator of the original board is set as an admin on the board, with the ability to change the board’s permissions. Other members are granted the **Editor** role.
 
+.. note::
+  
+  There are schema changes in v7.2. If you have a large database you may need to allocate additional time and resources to this migration. Please review `this document <https://docs.google.com/document/d/1hKQV6QfXSJ6LznvKxoENvU1Z21ulaZ09b-h8N1caS64/edit#heading=h.1ozc9hcotgot>`_ for details around permission migrations.
```

Comment:
```suggestion
  There are schema changes introduced in Mattermost v7.2. If you have a large database, you may need to allocate additional time and resources to this migration. Please review `this document <https://docs.google.com/document/d/1hKQV6QfXSJ6LznvKxoENvU1Z21ulaZ09b-h8N1caS64/edit#heading=h.1ozc9hcotgot>`_ for details around permission migrations.
```

Reply:
I'd use `Boards` instead of `Mattermost` here, as those changes are specific to the Boards plugin and they don't affect Mattermost Channels database schema